Makefile

```diff
@@ -123,9 +123,12 @@ test-dev-integration test-ci-integration: .init-swarm _check_venv_active## Run i
 	export DOCKER_IMAGE_TAG=production; \
 	make --no-print-directory _run-$(subst -integration,,$@) target=$(CURDIR)/tests/integration
 
-test-dev-system test-ci-system: ## Run integration tests.
+test-dev-system: ## Run integration tests.
+	@./ci/github/system-testing/deployment-agent.bash install && \
```

Comment:
you really want to install each time you run the system tests?
And actually why is the Makefile having dependencies on the github scripts? that is strange.

Reply:
TIP: you also create pipelines with dependencies in Makefile